math/mathcore/inc/Fit/FitUtil.h

```diff
@@ -784,9 +789,197 @@ namespace FitUtil {
          return -1.;
       }
 
-      static void EvalChi2Gradient(const IModelFunctionTempl<T> &, const BinData &, const double *, double *, unsigned int)
+      // Compute a mask to filter out infinite numbers and NaN values.
+      // The argument rval is updated so infinite numbers and NaN values are replaced by
+      // maximum finite values (preserving the original sign).
+      static vecCore::Mask<T> CheckInfNaNValues(T &rval)
```

Comment:
Yeah, it's a huge chunk of code inside the `ifdef`. Thanks for the review, you reminded me that I should also build without `veccore`, I was not doing it